ephios/settings.py

```diff
@@ -196,26 +196,43 @@
 ADMINS = getaddresses([env("ADMINS")])
 
 # logging
-if not DEBUG:
-    LOGGING = {
-        "version": 1,
-        "disable_existing_loggers": True,
-        "handlers": {
-            "mail_admins": {"level": "ERROR", "class": "django.utils.log.AdminEmailHandler"}
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "default": {"format": "%(levelname)s %(asctime)s %(name)s %(module)s %(message)s"},
+    },
+    "filters": {
+        "require_debug_false": {
+            "()": "django.utils.log.RequireDebugFalse",
         },
-        "loggers": {
-            "django": {
-                "handlers": ["mail_admins"],
-                "level": "ERROR",
-                "propagate": False,
-            },
+        "require_debug_true": {
```

Comment:
do we need this one?

Reply:
not the `true` one, but I included both for completeness...